### DISCUSSION TITLE
R9 Stab OTA, support

### DISCUSSION THREAD

**Discussion by frankberliner** [ExpressLRS/ExpressLRS#1353] [Ideas]
Hi, Does this model support ExpressLRS?

https://www.frsky-rc.com/product/r9-stab-ota/

**Comment by greenbigfrog** [comment]
At the moment, probably no. It would probably be rather simple to just add a target (for someone more experienced), although you don't have much benefit from it, since you'll loose stab as well as servo output since eLRS doesn't support PWM (yet).

**Comment by pHASShon** [comment]
I'll be glad to help on some testing with this RX

**Comment by jpmarotta** [comment]
I've got this RX too, would be able to test as well.
